### DISCUSSION TITLE
Remove Keka

### DISCUSSION THREAD

**Discussion by hstriepe** [aonez/Keka#1112] [Q&A]
I want to remove Keka from being the default compressor/decompressor and have removed the app completely. I have searched for and deleted containers, App Support, etc. I have deleted Finder Preferences.
Yet the Keka icon keeps showing as the default for archives.

I only want to use Keka by choice to avoid potential conflicts during development.

**Comment by aonez** [comment]
Do this:

https://github.com/aonez/Keka/wiki/Default-application#set-it-manually

Just set whatever app you want as default there. Even if the icon show, it might not be the default app, macOS is very inconsistent with icons lately.

**Reply by hstriepe** [reply]
You mean RTFM? I did. I still have the red icon. Everything I could find is deleted and the Archive Utility is set as default. I relaunched the Finder, rebooted, etc. Still red!
Monterey 12.4 on M1 Ultra.

**Reply by aonez** [reply]
RTFM?
Sadly this is an issue within macOS, here the inverse: https://github.com/aonez/Keka/discussions/1105

You can try clearing the icons cache as stated here: https://github.com/aonez/Keka/discussions/1105#discussioncomment-2818410

**Reply by hstriepe** [reply]
RTFM? Actual Read The F???? Web Page, not Manual. 😀

That worked! Thanks!

I have been battling sudden failures to notarize and wanted to make sure, there is nothing in the compression for upload that is unexpected to the Apple house of cards. spctl -a -vv is happy but it still fails signature check. You know the drill.

**Reply by aonez** [reply]
> RTFM? Actual Read The F???? Web Page, not Manual. 😀

😂

Good luck with the notarization. I've recently had one of those, turned out one of the bundled binaries wasn't properly signed.

**Reply by hstriepe** [reply]
We have a lot of those. Combing through. Adding Universal support can be a pain for complex apps.
The good news: our last release still notarizes. Soooo...
